treeherder/seta/management/commands/reset_seta.py

```diff
@@ -1,13 +1,18 @@
 from django.core.management.base import BaseCommand
```

Comment:
is there a way we could determine if there is no data (a fresh install) that we reset everything so it is properly setup?